.gitpod.yml

```diff
@@ -9,11 +9,13 @@ tasks:
       eval $(gp env -e API_URL=$(gp url 3000))
       eval $(gp env -e WEB_URL=$(gp url 8000))
       eval $(gp env -e API_HOST=$(gp url 8443))
+      eval $(gp env -e SSO_URL=$(gp url 8081))
       eval $(gp env -e SERVICES="status image auth search posts feed-discovery users redis elasticsearch login firebase")
 
       sed -r \
           -e  "s@(.+=)http://localhost:8000(/[^ ]*)*@\1$WEB_URL\2@g" \
           -e  "s@(.+=)http://localhost:3000(/[^ ]*)*@\1$API_URL\2@g" \
+          -e  "s@(.+=)http://localhost:8081(/[^ ]*)*@\1$SSO_URL\2@g" \
```

Comment:
I was just surprised to see this change, since it is based on my other PR, right?

Reply:
No, I just happened to have the same idea.